src/net/chat.rs

```diff
@@ -175,7 +179,7 @@ pub struct ConversationP2P {
     pub peer_heartbeat: u32,                    // how often do we need to ping the remote peer?
     pub peer_expire_block_height: u64,          // when does the peer's key expire?
 
-    pub data_url: UrlString,                   // where does this peer's data live?
+    pub data_url: UrlString,                   // where does this peer's data live?  Set to a 0-length string if not known.
```

Comment:
Why not an Optional?

Reply:
It's just cosmetic.  The URL won't be used either way.